platform/security/handler/security-handler-saml/src/main/java/org/codice/ddf/security/idp/client/LogoutRequestService.java

```diff
@@ -580,16 +572,14 @@ private void logout() {
 
   private void logSecurityAuditRole() {
     Element idpSecToken = getIdpSecurityToken();
-    if (idpSecToken != null) {
-      if (shouldAuditSubject(idpSecToken)) {
-        securityLogger.audit(
-            "Subject with admin privileges has logged out: {}",
-            new SecurityAssertionSaml(idpSecToken).getPrincipal().getName());
-      }
+    if (idpSecToken != null && shouldAuditSubject(idpSecToken)) {
+      securityLogger.audit(
+          "Subject with admin privileges has logged out: {}",
+          new SecurityAssertionSaml(idpSecToken).getPrincipal().getName());
     }
   }
 
-  private boolean shouldAuditSubject(Element idpSecToken) {
+  private static boolean shouldAuditSubject(Element idpSecToken) {
```

Comment:
❓ Why static (same question for the build logout response methods)?

Reply:
That's just a style preference. I like using `static` for helper methods because it indicates that no instance variables are being used.